ShopifySharp/Filters/IListFilter.cs

```diff
@@ -14,6 +15,11 @@ public interface IListFilter
         /// </summary>
         int? Limit { get; set; }
 
+        /// <summary>
+        /// A comma-separated list of which fields to show in the results. This parameter only works for some endpoints.
+        /// </summary>
+        string Fields { get; set; }
```

Comment:
I think I want to leave `Fields` off of the base list filter, as not all of the endpoints using pagination make use of the `Fields` property. For v5 I'm going more with the philosophy of creating dedicated list filters for each service, and the dedicated ones add the properties (like fields, created_at_min, etc.) that the service supports. 

In v4 I went a little too overboard with DRY and tried to make every filter extend other filters as much as possible, even if the endpoints didn't support some of the properties. I want to get away from that in v5.

Reply:
I think this new philosophy is the right way to go, however Shopify does clearly state that 3 parameters only can be changed when paginating: page_info, limit and fields.
https://shopify.dev/tutorials/make-paginated-requests-to-rest-admin-api

fields is unlikely to be changed from one page to the next though. I will remove it.

Reply:
I removed the Fields property.